app/Http/Controllers/MemberController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\StoreCreateMember;
+use App\Member;
+use App\User;
+use App\Position;
+use Validator;
+use DB;
+
+class MemberController extends Controller
+{
+    public function listMember(Request $request)
+    {
+        $listMember= Member::all()->toArray();
+        dd($listMember) ;
+    }
+
+    public function deleteMember($id)
+    {
+        $member=Member::find($id);
+        $member ->delete();
+        return "xóa thành công" ;
+    }
+
+    public function getCreateMember()
+    {
+        return view('test');
+    }
+
+    public function getEditMember($id)
+    {
+        $item = Member::find($id)->toArray();
+        return view('test', ['id'=>$id, 'item'=>$item]);
+    }
+
+    public function editMember(StoreCreateMember $request)
+    {
+        $data=$request->all();
+        $memberEdit=Member::find($data['id']);
+        $memberEdit->name=$data['name'];
+        $memberEdit->phone_number=$data['phone_number'];
+        $memberEdit->information=$data['information'];
+        $memberEdit->birthday=$data['birthday'];
+        $memberEdit->position_id=$data['position_id'];
+        $memberEdit->position = DB::table('positions')
```

Comment:
We just need Position_id field dont need position field. Pls remove Position field.